build.gradle

```diff
@@ -18,7 +18,7 @@ ext.scenariooAggregatedDataFormatVersion = '2.1.0'
 /**
  * Name of the release branch of this scenarioo version, which is used for links to the versioned documentation.
  */
-ext.scenariooReleaseBranch = 'release-3.0'
+ext.documentationVersion = '3.0'
```

Comment:
Shouldn't this point to 'develop'?

Reply:
No - this is the link for current product version - all our links in our product point to the documentation for the current installed version - once we do the release we will increase that to 4.0 and deploy the new docu.